bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableConfiguration.java

```diff
@@ -44,7 +44,7 @@
   }
 
   public static Configuration configure(String projectId, String zoneName, String clusterName) {
-    Configuration config = new Configuration();
```

Comment:
my question was is this configure() in BigtableConfiguration.java for dataflow only?